linux_docker_resources/Dockerfile

```diff
@@ -44,8 +43,6 @@ RUN curl --silent http://packages.osrfoundation.org/gazebo.key | apt-key add -
 # Install some development tools.
 RUN apt-get update && apt-get install --no-install-recommends -y build-essential ccache cmake pkg-config python3-empy python3-pip python3-setuptools python3-vcstool python3-venv
 RUN apt-get update && apt-get install --no-install-recommends -y python3-lark python3-opencv
-# Install virtualenv 16.7.9 needed for Foxy builds on Focal. https://github.com/ros2/ci/issues/400
-RUN if test ${UBUNTU_DISTRO} = focal; then python3 -m pip install virtualenv==16.7.9; fi
```

Comment:
Glad to see the back of this kludge!

Reply:
We still have it in place in https://github.com/ros2/ci/blob/2c8cddea04fd79c1e5d8805a133e695d6f724c52/ros2_batch_job/__main__.py#L472 .  Do you think I should attempt to remove it from there as well?

Reply:
I'm going to go forward without this for now.  We can always revisit it in a separate PR.